.eslintignore

```diff
@@ -40,6 +40,7 @@ bower_components
 /x-pack/legacy/plugins/infra/public/graphql/types.ts
 /x-pack/legacy/plugins/infra/server/graphql/types.ts
 /x-pack/legacy/plugins/apm/cypress/**/snapshots.js
+/src/legacy/plugin_discovery/plugin_pack/__tests__/fixtures/plugins/broken
```

Comment:
The globbing has been improved a little so the code in this directory was being tested now.